### DISCUSSION TITLE
Reduce font-size of H1 in page

### DISCUSSION THREAD

**Discussion by fesja** [outline/outline#3974] [Ideas]
The font size of the page title and the H1 headings is the same (36px). This provokes some visual inconsistency, as the page title should be bigger than its headings.

I suggest we make H1 headings a little bit smaller (2em instead of 2.25em, 32px).

**Comment by tommoor** [comment]
Ah yes, I was thinking the same thing just yesterday – may make the page titles just a touch larger too.
